common/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -608,6 +608,7 @@ message LoadJobPOptions {
   optional bool partialListing = 3;
   optional bool loadMetadataOnly = 4;
   optional bool skipIfExists = 5;
+  optional int32 replicas = 6;
```

Comment:
im assuming this is not used in this PR right?

Reply:
would be used in following PR